95_windows/20_agent_installation.txt

```diff
@@ -2,6 +2,17 @@
 === Install Windows DSC agent
 
 The installation and upgrade processes are exactly the same.
+The Rudder agent needs *PowerShell 4* or later to be functionnal, which is built-in on:
+
+ * Windows Server 2012 R2 and later
+
+And Available on:
+
+ * Windows Server 2008 R2
```

Comment:
install doc is there: https://social.technet.microsoft.com/wiki/contents/articles/20623.step-by-step-upgrading-the-powershell-version-4-on-2008-r2.aspx